zulipterminal/ui_tools/boxes.py

```diff
@@ -710,6 +715,7 @@ def autocomplete_emojis(
         return emoji_typeahead, emojis
 
     def exit_compose_box(self) -> None:
+        self._set_default_footer_after_autocomplete()
```

Comment:
Having extracted this function in the earlier commit, it would be useful to confirm why we need it here in the commit body.